concepts/05 Widgets/ColorBox/15 Support Alpha Channel.md

```diff
@@ -33,6 +33,50 @@ By default, the **ColorBox** does _not_ allow an end user to control the transpa
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxColorBox
+            value="rgba(255, 144, 0, 0.3)"
```

Comment:
Please use two-way binding as in the Angular approach.

Reply:
fixed